server/src/test/scala/za/co/absa/atum/server/api/database/runs/functions/CreatePartitioningIfNotExistsSpec.scala

```diff
@@ -17,6 +17,7 @@
 package za.co.absa.atum.server.api.database.runs.functions
 
 import org.junit.runner.RunWith
+import shapeless.tag
```

Comment:
This import is not used and should be removed.

Reply:
Addressed.